torch/quantization/quantize.py

```diff
@@ -169,46 +173,53 @@ def __init__(self):
     def forward(self, x):
         return x
 
-def quantize(module, eval_fn, eval_args, qconfig_dict=None):
-    r"""Converts a float module to quantized module.
+def quantize(model, eval_fn, eval_args, qconfig_dict=None):
+    r"""Converts a float model to quantized model.
 
-    First it will prepare the module for calibration or training, then it calls
+    First it will prepare the model for calibration or training, then it calls
     `eval_fn` which will run the calibration step or training step,
-    after that we will call `convert` which will convert the module to a
-    quantized module.
+    after that we will call `convert` which will convert the model to a
+    quantized model.
 
     When `qconfig_dict` is None or empty dictionary, we will assume user will
     insert quant/dequant stubs and add qconfig in approporiate places.
     When `qconfig_dict` is not None or empty dictionary, we will add quant/dequant
     stubs using QuantWrapper for all the leaf modules.
 
     Args:
-        module: input module
-        eval_fn: a function for evaluating the prepared module, can be a
-            function that simply runs the prepared module or a training loop
+        model: input model
+        eval_fn: a function for evaluating the prepared model, can be a
+            function that simply runs the prepared model or a training loop
         eval_args: positional arguments for `eval_fn`
         qconfig_dict: dictionary that maps from name of submodule to quantization
             configuration, qconfig applies to all submodules of a given
-            module unless qconfig for the submodules are specified(when the
+            model unless qconfig for the submodules are specified(when the
             submodule already has qconfig attribute)
 
 
     Return:
-        A quantized module
+        A quantized model
     """
-    module = prepare(module, qconfig_dict)
-    eval_fn(module, eval_args)
-    convert(module)
-    return module
+    model = prepare(model, qconfig_dict)
+    eval_fn(model, eval_args)
```

Comment:
Lets call this run_model instead of eval or training so that it is generic for both flows.

Reply:
changing to `run_fn` and `run_args`